region/maridia/outer/Mt. Everest.json

```diff
@@ -2113,6 +2114,28 @@
       ],
       "note": "After entering the room, quickly leave the morph tunnel in order to prevent getting hit by the crab."
     },
+    {
+      "link": [6, 11],
+      "name": "Shinespark Deep Transition X-Ray Climb",
+      "entranceCondition": {
+        "comeInWithSpark": {
+          "position": "top"
+        }
+      },
+      "requires": [
+        "canDeepTransition",
```

Comment:
I don't have time to review this PR right now, but does this strat need `h_EverestMorphTunnelExpanded`?

Reply:
I think it should be ok without it. With the tunnel not expanded, the strat still works, and in fact you don't even need a deep transition; any top-position shinespark would work. If a different project did something different, like expanding the height of the entire tunnel instead of just the entrance, then the strat wouldn't work.